src/settings/SettingsSelfExclusion.js

```diff
@@ -68,13 +68,19 @@ export default class SettingsSelfExclusion extends PureComponent {
 			exclude_until,
 			timeout_until,
 		};
+
 		try {
 			await api.setSelfExclusion(newSelfExclusionSettings);
 			this.setState({
 				success: true,
 				serverError: false,
 			});
-			setTimeout(() => this.setState({ success: false }), 3000);
+			setTimeout(() => {
```

Comment:
@borisyankov  this is what i am thinking. Please see the update and here https://github.com/binary-com/binary-live-api/pull/70/files